test/unit/server/handlers/participants/{Type}/{ID}.test.ts

```diff
@@ -24,10 +24,12 @@
  - Kevin Leyow <kevin.leyow@modusbox.com>
  --------------
  ******/
-import { Request, ResponseToolkit } from '@hapi/hapi'
+import { Request } from '@hapi/hapi'
 import { Enum } from '@mojaloop/central-services-shared'
-import { h } from 'test/data/data'
 import ParticipantsTypeIDHandler from '~/server/handlers/participants/{Type}/{ID}'
+import { StateResponseToolkit } from '../../../../../../src/server/plugins/state';
```

Comment:
Any reason to use these instead of the `~`?

Reply:
auto import in vscode imports them like this and I sometimes miss it. 
been meaning to find a way to fix that.